### ISSUE TITLE
feat (Labware): Labware creator testing file should include instructions

### ISSUE THREAD

**Issue by alexjoel42** [Opentrons/opentrons#10843]
# Overview

The Labware Testing guide is hard to follow with the Python protocol since the steps aren't labeled. 

# Implementation details
1. Modify the Python files that come with custom labware defintions
2. Include the steps in the Labware Definition Test guide 

# Design

# Acceptance criteria

- Each pause has an explanation of what is happening
- Someone should be able to walk into a lab where a Labware Definition test is occurring and be able to identify which step the robot is on. 
<img width="687" alt="Screen Shot 2022-06-21 at 11 30 10 AM" src="https://user-images.githubusercontent.com/65437763/174838891-c96b266f-fa79-4091-80ed-e9da7912545d.png">

**Comment by mikecardarelli** [comment]
@skyewindsor, Just getting this on your radar for the next time we're making updates in LC

**Comment by mikecardarelli** [comment]
New JIRA ticket: https://opentrons.atlassian.net/browse/RAUT-200
